packages/block-editor/src/hooks/align.native.js

```diff
@@ -28,9 +23,11 @@ addFilter(
 			settings.supports = {
 				...settings.supports,
 				align: Array.isArray( blockAlign )
-					? without(
-							blockAlign,
-							...Object.values( WIDE_ALIGNMENTS.alignments )
+					? blockAlign.filter(
+							( alignment ) =>
+								! Object.values(
```

Comment:
Quick note that this will run `Object.values` on the same object on every iteration of the loop, but I don't expect that to be a hot path.